### PR TITLE
Enable updater for windows

### LINKED ISSUE #1789: Updater is accidentially left disabled on Windows
We added windows support to the updater but it's still left disabled.

### PR DESCRIPTION
**Pull request by dominikschulz**
Fixes #1789

GOPASS_FORCE_UPDATE can now also override the version check.

RELEASE_NOTES=[BUGFIX] Enable updater on Windows

Signed-off-by: Dominik Schulz <dominik.schulz@gauner.org>

### CODE REVIEW

File: internal/updater/updateable.go [hunk 1]
```diff
@@ -46,13 +42,22 @@ func IsUpdateable(ctx context.Context) error {
...
+	// no need to check the directory since we'll be writing to the destination file directly
+	return nil
+}
+
+func canWrite(path string) error {
+	// check if we can write to this file. we open it for writing in
+	// append mode to not truncate it and immediately close the filehandle since
+	// we're only interested in the error.
+	fh, err := os.OpenFile(path, os.O_WRONLY|os.O_APPEND, 0755)
+	fh.Close()
```
Comment (AnomalRoil) [inline internal/updater/updateable.go:58]: I would check the error on `Close`, just in case.

**Review by AnomalRoil — APPROVED** [review]

**Comment by AnomalRoil** [comment]
I'm not sure what's happening with the tests, strange.

File: internal/updater/updateable.go (hunk 1 above)
Comment (dominikschulz) [reply internal/updater/updateable.go:58]: Hmm, can `Close()` even fail when not writing anything to the file handle?

**Comment by dominikschulz** [comment]
Ah, I see what's wrong with the tests.
That looks like a pre-existing bug that was uncovered by my changed to the progress bar.
Will fix them as part of this PR.

**Review by AnomalRoil — APPROVED** [review]

**Pull request merged**